modules/041-linstor/templates/linstor-scheduler/deployment.yaml

```diff
@@ -100,7 +100,7 @@ spec:
             valueFrom:
               fieldRef:
                 fieldPath: metadata.namespace
-          image: {{ include "helm_lib_module_image" (list . (list "kubeScheduler" $kubeVersion.Major $kubeVersion.Minor | join "")) }}
+          image: {{ include "helm_lib_module_image" (list (dict "Chart" (dict "Name" "control-plane-manager") "Values" .Values) (list "kubeScheduler" $kubeVersion.Major $kubeVersion.Minor | join "")) }}
```

Comment:
Could you add a comment here please, which explain what happens here :)

Reply:
Added